pkg/kubelet/rkt/container_id.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2015 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rkt
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/types"
+)
+
+// containerID defines the ID of rkt containers, it will
+// be returned to kubelet, and kubelet will use this for
+// container level operations.
+type containerID struct {
+	uuid    string // uuid of the pod.
+	appName string // name of the app in that pod.
+	imageID string // id of the image. TODO(yifan): Depreciate this.
+}
+
+// buildContainerID constructs the containers's ID using containerID,
+// which containers the pod uuid and the container name.
+// The result can be used to globally identify a container.
+func buildContainerID(c *containerID) types.UID {
```

Comment:
should this return a string? I don't think it's a UID as the types.UID is a UID.